LabExT/View/Movement/CoordinatePairingsWindow.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+LabExT  Copyright (C) 2022  ETH Zurich and Polariton Technologies AG
+This program is free software and comes with ABSOLUTELY NO WARRANTY; for details see LICENSE file.
+"""
+
+from tkinter import Frame, Toplevel, Button, Label, messagebox, LEFT, RIGHT, TOP, X, BOTH, DISABLED, FLAT, NORMAL, Y
+from LabExT.View.Controls.CustomFrame import CustomFrame
+from LabExT.View.Controls.DeviceTable import DeviceTable
+
+from LabExT.Movement.Stage import StageError
+import LabExT.Movement.Transformations as Transformations
+
+
+class CoordinatePairingsWindow(Toplevel):
```

Comment:
I am wondering where you enforce that only one coordinate pairing (as its single point fixed) can be done. Is this inherent to this window, so it will only allow a single point per attached stage?